include/robot_localization/ros_filter.h

```diff
@@ -255,6 +258,19 @@ template<class T> class RosFilter
                        const double mahalanobisThresh);
 
   protected:
+    //! Finds the latest filter state before the given timestamp and makes it
```

Comment:
Just for consistency, can we duplicate the other methods and do
//! `@`brief brief description
//! ...longer description...
//! `@`param[in] t Description of t